test/Table_Tests/src/Common_Table_Operations/Map_To_Table_Spec.enso

```diff
@@ -0,0 +1,83 @@
+from Standard.Base import all
+import Standard.Base.Errors.Common.Assertion_Error
+import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+
+from Standard.Database.Data.Table import MAX_LITERAL_ELEMENT_COUNT
+
+from Standard.Test_New import all
+
+from project.Common_Table_Operations.Util import run_default_backend
+
+main = run_default_backend add_specs
+
+type Data
+    Value ~data
+
+    connection self = self.data.at 0
+    dummy_table self = self.data.at 1
+
+    setup test_setup create_connection_fn =
+        connection = (create_connection_fn Nothing)
+        dummy_table = test_setup.table_builder [['x', [1]]] connection=connection
+        Data.Value [connection, dummy_table]
+
+    teardown self = self.connection.close
+
+
+add_specs suite_builder setup =
+    prefix = setup.prefix
+    create_connection_fn = setup.create_connection_func
+    suite_builder.group prefix+"Table.make_table_from_map/vectors" group_builder->
+        data = Data.setup setup create_connection_fn
+
+        group_builder.teardown <|
+            data.teardown
+
+        group_builder.specify "should be able to create a literal table with one column" <|
+            vecs = [[0], [3], [6]]
+            t = data.dummy_table.make_table_from_vectors vecs ['x', 'y', 'z']
+            t.at 'x' . to_vector . should_equal [0]
+            t.at 'y' . to_vector . should_equal [3]
+            t.at 'z' . to_vector . should_equal [6]
+
+        group_builder.specify "should be able to create a literal table with three columns" <|
+            vecs = [[0, 1, 2], [3, 4, 5], [6, 7, 8]]
+            t = data.dummy_table.make_table_from_vectors vecs ['x', 'y', 'z']
+            t.at 'x' . to_vector . should_equal [0, 1, 2]
+            t.at 'y' . to_vector . should_equal [3, 4, 5]
+            t.at 'z' . to_vector . should_equal [6, 7, 8]
+
+        group_builder.specify "should not be able to create a literal table with empty columns" <|
+            data.dummy_table.make_table_from_vectors [[], [], []] ['x', 'y', 'z'] . read . should_fail_with Illegal_Argument
```

Comment:
Given this test, I'd like to also see the behaviour of `Table.replace` tested with the edge case when the replace table/map (2 variants) is empty.

Reply:
Done